runtimevar/constantvar/constantvar_test.go

```diff
@@ -17,6 +17,7 @@ package constantvar
 import (
 	"context"
 	"errors"
+	"github.com/google/go-cmp/cmp"
```

Comment:
nit: This belongs in the second import group (non-stdlib).

Reply:
Done.